USF-1911: Restructured PaymentMethods container with deprecated configs (#181)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/checkout/containers/payment-methods.mdx

CheckoutProvider.render(PaymentMethods)($paymentMethods),
```

> ## Example 2: How to use the `displayLabel` and `icon` configurations

Reviewer: ```suggestion
## Example 2: Render with the `displayLabel` and `icon` configurations
```

Author: Done

The following example renders the `PaymentMethods` container on a checkout page, displaying the available payment methods in the element with the class `checkout__payment-methods`, providing an icon for `checkmo` and `banktransfer`, and hiding the label for `banktransfer`:

```ts
// Checkout Dropin
import PaymentMethods from '@dropins/storefront-checkout/containers/PaymentMethods.js';
import { render as CheckoutProvider } from '@dropins/storefront-checkout/render.js';

const $paymentMethods = checkoutFragment.querySelector(
  '.checkout__payment-methods',
);

CheckoutProvider.render(PaymentMethods, {
  slots: {
    Methods: {
      checkmo: {
        icon: 'Wallet',
        render: (ctx) => {
          const $content = document.createElement('div');

          $content.innerText = 'Pay later with Checkmo config handler';
          ctx.replaceHTML($content);
        },
      },
      banktransfer: {
        displayLabel: false,
        icon: 'Card',
      },
    },
  },
})($paymentMethods),

```

> ## Example 3: How to use the `setOnChange` and `render` configurations

Reviewer: ```suggestion
## Example 3: Render with the `setOnChange` and `render` configurations
```

Author: Done

let braintreeInstance;

CheckoutProvider.render(PaymentMethods, {
  slots: {
    Methods: {
      braintree: {
        setOnChange: false,
        render: async (ctx) => {
          const container = document.createElement('div');

          window.braintree.dropin.create({
            authorization: 'sandbox_cstz6tw9_sbj9bzvx2ngq77n4',
            container,
          }, (err, dropinInstance) => {
            if (err) {
              console.error(err);
            }

            braintreeInstance = dropinInstance;
          });

          ctx.replaceHTML(container);
        },
      },
    },
  },
})($paymentMethods),

```

> ## Example 4: How to use the `enabled` configurations

Reviewer: ```suggestion
## Example 4: Render with the `enabled` configurations
```

Author: Done
